src/languages/golfscript/index.ts

```diff
@@ -87,7 +87,13 @@ const golfscriptLanguage: Language = {
 
       ["argv_get", "a="],
     ]),
-    addImports,
+    addImports(
+      [
+        ["a=", "a"],
+        ["a", "a"],
+      ],
+      (x) => (x.length > 0 ? assignment(x[0], id("", true)) : undefined)
+    ),
```

Comment:
What does this do? My guess is it sets up an import for any usage of `a` or `a=`, and the import would be an assignment of `a` to be the empty identifier, so it'll look like `:a`. Oh, is this loading the arguments stack into a variable?

Reply:
> What does this do? My guess is it sets up an import for any usage of `a` or `a=`, and the import would be an assignment of `a` to be the empty identifier, so it'll look like `:a`. Oh, is this loading the arguments stack into a variable?

Thats's correct.